documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -1446,6 +1446,13 @@ twice - once in initial snapshot and once in streaming phase. Nonetheless, that
 data mirroring.
 For `read_uncommitted` there are no data consistency guarantees at all (some data might be lost or corrupted).
 
+|`event.processing{zwsp}.failure.handling.mode`
+|`fail`
+| Specifies how the connector should react to exceptions during processing of events.
```

Comment:
> indicating the problematic event and its offset

Actually only the offset is revealed, which I think is good. So the comment should be adjusted. Same for the enum JavaDoc.